cdap-docs/developer-manual/source/cloud-runtimes/provisioners/aws-emr.rst

```diff
@@ -8,5 +8,69 @@
 Amazon Elastic MapReduce
 ========================
 
-Describe what EMR is, then describe each config setting
+Elastic MapReduce is an Amazon Web Services product, which provides a managed cluster platform for running
+big data processing and analysis on frameworks such as Apache Hadoop and Apache Spark.
+The Amazon EMR provisioner simply calls the EMR APIs in order to create and delete
+clusters in your AWS account.
+The provisioner exposes several configuration settings that control what type of cluster is created.
 
+Account Information
+-------------------
+
+Access Key
+^^^^^^^^^^
+An AWS Access Key ID identifies an AWS access key, which can be used to make secure REST or HTTP query
+protocol requests to AWS service APIs.
+
+Secret Key
+^^^^^^^^^^
+An AWS Secret Key can be used to make secure REST or HTTP query protocol requests to AWS service APIs.
+Since your secret key is sensitive, it is recommended that you
+provide your secret key through the CDAP :ref:`Secure Storage <http-restful-api-secure-storage>` API.
```

Comment:
let's use the same language as in the other provisioners (now merged to release branch)